stories/Who_was_protected_by_the_first_COVID-19_lockdown.ipynb

```diff
@@ -0,0 +1,2412 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-10-07T15:26:11.316228Z",
+     "start_time": "2020-10-07T15:26:10.154042Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import requests\n",
+    "from datetime import datetime\n",
+    "import zipfile\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from scipy.stats import  pearsonr\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "sns.set(style='white')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "🔧 Let's make some directories in which we can store the data we are going to download. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-10-07T15:26:11.321873Z",
+     "start_time": "2020-10-07T15:26:11.318558Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# downloaded data goes here\n",
+    "downloaded_data_dir = 'data/downloaded'\n",
+    "\n",
+    "# processed data goes here\n",
+    "derived_data_dirname = 'data/derived'\n",
+    "\n",
+    "# create the directory if it does not already exist\n",
+    "os.makedirs(downloaded_data_dir, exist_ok=True)\n",
+    "os.makedirs(derived_data_dirname, exist_ok=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "🔧 Here is a small helper function which will download files from a URL."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2020-10-07T15:26:11.327866Z",
+     "start_time": "2020-10-07T15:26:11.324483Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# This function can download data from a URL and then save it in a directory of our choice.\n",
+    "def download_file(url, filename):\n",
+    "        \n",
+    "    # create the directory if it does not already exist\n",
+    "    os.makedirs(os.path.dirname(filename), exist_ok=True)\n",
+    "\n",
+    "    # make the HTTP request\n",
+    "    r = requests.get(url, allow_redirects=True)\n",
+    "\n",
+    "    # save file\n",
+    "    _ = open(filename, 'wb').write(r.content)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Index of Multiple Deprivation (IMD)\n",
+    "\n",
+    "🔧 Now let's download and process our deprivation data. This data provides a deprivation rank (lower rank meaning more deprived) for each geographical region in England (the geographical regions are here are called Lower Super Output Areas, or [LSOAs](https://www.datadictionary.nhs.uk/nhs_business_definitions/lower_layer_super_output_area.html#:~:text=Description,statistics%20in%20England%20and%20Wales.))."
```

Comment:
A brief note on the typical population of an LSOA would be really useful here (the link does provide it, but it takes a bit of reading of the page).